query-engine/connector-test-kit-rs/query-engine-tests/tests/new/regressions/prisma_14696.rs

```diff
@@ -0,0 +1,75 @@
+use query_engine_tests::*;
+
+#[test_suite(schema(schema))]
```

Comment:
The idea with this test was to stick as close as possible to the issue. It could be more minimal.